src/components/garagePromotionCard.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { FC, MouseEvent } from "react"
+
+import UploadImagesIcon from "./icons/uploadImage"
+import ArrowRightTeal from "./icons/arrowRight"
+import { FixedCard } from "./card/fixed"
+
+interface Props {
+  dealerName: string
+  dealerLocation: {
+    address?: string
+    zipCode: string
+    city?: string
+  }
+  title?: string
+  previewLabel: string
+  mainImage?: string
```

Comment:
are there any secondary images? or can this just be called `image`?
I would consider making this mandatory too. it will not look any good without an image I'd say...

Reply:
No secondary image. I will make it mandatory 👍